app/templates/message_template.hbs

```diff
@@ -1,34 +1,36 @@
 <section class="message-section">
   <div class="row">
     <div class="small-12 columns">
-      {{#each groupedElements as |messages|}}
-        <div class="small-12 columns day_seperator">
-          <span>{{display-messages-date messages.firstObject.createdDate (t 'day.today')}}</span>
-        </div>
-        {{#each messages as |message|}}
-          {{#if message.isMessage}}
-            <div class="{{message.state}} {{if message.myMessage 'my_message' 'received_message'}}" id={{message.id}}>
-              <div class='message_details'>
-                {{#if message.myMessage}}
-                  {{t "messages.owner" }}
-                {{else}}
-                  {{custom-img-tag src=message.sender.displayImageUrl class="user-avatar"}}
-                  {{t "full_name" firstName=message.sender.firstName lastName=message.sender.lastName}}
-                  {{message.sender.roleInitials}}
-                {{/if}}
-                →
-                {{display-datetime message.createdAt format="HH:mm"}}
-              </div>
+      {{#href-to-link-to}}
```

Comment:
@swatijadhav This component is wrapping all messages underneath it. Is there a reason why we don't run the component on each message individually?

cc @mcm-ham